### ISSUE TITLE
Toolbar: aria-label

### ISSUE THREAD

**Issue by ethanolree** [primefaces/primeng#16173]
### Describe the bug

Accessibility documentation for the Toolbar shows the use of the aria-label. However, this provides no value unless an input for aria-label is provided so that the aria-label can be placed where the toolbar role is. 

![image](https://github.com/user-attachments/assets/ca4f1e10-a0ca-44ce-be31-57b04b3a8a0f)

![image](https://github.com/user-attachments/assets/9231336a-7787-4f90-8b75-5db93cea6add)

### Environment

Angular Primeng

### Angular version

18.0.1

### PrimeNG version

17.18.7

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20

### Expected behavior

Should be able to make use of aria-label on the toolbar component

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation and try the latest PrimeNG version(v19). If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
